app/views/homepage/_government_activity.erb

```diff
@@ -0,0 +1,75 @@
+<!--
+Created a custom section and header pattern to
+- display a border-top of 2px (to replace .home-divider)
+- margin-bottom of 30px (both mobile and desktop)
+- padding-top to create a gap between border and heading
+-->
+<section class="homepage-section homepage-section__government-activity">
+  <div class="govuk-grid-row">
+    <div class="govuk-grid-column-two-thirds" data-module="gem-track-click">
+      <div class="homepage-section__heading">
+        <%= render "govuk_publishing_components/components/heading", {
+          text: t("homepage.index.government_activity"),
+          margin_bottom: 2,
+        } %>
+
+        <p class="govuk-body-l">
+          <%= t("homepage.index.government_activity_description") %>
```

Comment:
This could be a [lead paragraph with a custom margin bottom of 0](https://components.publishing.service.gov.uk/component-guide/lead_paragraph/custom_margin_bottom)

Reply:
Nice one, completely forgot I could use this as a component.